backend/server.js

```diff
@@ -0,0 +1,38 @@
+/* eslint no-console: 0 */
+
+import express from 'express'
+import bodyParser from 'body-parser'
+import cors from 'cors'
+import { check, validationResult } from 'express-validator/check'
+
+const app = express()
+
+/* if (process.env.NODE_ENV !== 'test') {
+	app.use(express.logger())
+} */
+
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: false }))
```

Comment:
I don't know if there's any difference, but express also exports its own `json`/`urlencoded` middleware. We could potentially use those and drop a dependency.
```js
app.use(express.json())
app.use(express.urlencoded({ extended: false })
```

Reply:
express used to not be able to handle POST requests without body-parser but i just tried this in the other project i stole this from and it works, good call